### DISCUSSION TITLE
Mantine v6/v7

### DISCUSSION THREAD

**Discussion by gius** [refinedev/refine#4736] [General]
Hi, do you plan to update the Mantine package to the latest version?

**Comment by BatuhanW** [comment]
Hello @gius, we've finished a spike, pre-investigation task about the upgrade, while I can't promise any time, we'll be doing the upgrade for sure in the future.

**Reply by gius** [reply]
Sounds great, thank you!

Are you potentially open for PRs?

**Reply by BatuhanW** [reply]
Yeah, of course! We'd be more than happy!
